### PR TITLE
Implement landing page

### PR DESCRIPTION
**Pull request by Savetang19**

### CODE REVIEW

File: wongnung/apps.py [hunk 1]
```diff
@@ -1,6 +1,15 @@
 from django.apps import AppConfig
+from allauth.socialaccount.signals import pre_social_login
+
+
+def callback(sender, **kwargs):
+    print(sender)
+    print([*kwargs])
```
Comment (GToidZ) [inline wongnung/apps.py:7]: I forgot to remove this, XD
```suggestion
```

File: wongnung/apps.py [hunk 2]
```diff
@@ -1,6 +1,15 @@
...
 
 
 class WongnungConfig(AppConfig):
     default_auto_field = "django.db.models.BigAutoField"
     name = "wongnung"
+
+    def ready(self) -> None:
+        pre_social_login.connect(callback)
```
Comment (GToidZ) [inline wongnung/apps.py:15]: This too.
```suggestion
```

**Review by GToidZ — CHANGES_REQUESTED** [review]

**Review by Supakrit65 — APPROVED** [review]
Nice looks

**Review by GToidZ — APPROVED** [review]
Changes approved! You can merge now! 🎉

**Comment by Savetang19** [comment]
[Frontend: landing page](https://trello.com/c/Ochj87v0/48-frontend-landing-page)

**Pull request merged**